N/NEO/build_tarballs.jl

```diff
@@ -72,9 +72,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("gmmlib_jll"; compat="=21.1.3"),
-    Dependency("libigc_jll"; compat="=1.0.7423"),
-    Dependency("oneAPI_Level_Zero_Headers_jll"; compat="=1.1.2"),
+    Dependency("gmmlib_jll"; compat="=21.2.1"),
+    Dependency("libigc_jll"; compat="=1.0.7862"),
+    Dependency("oneAPI_Level_Zero_Headers_jll"; compat="~1.2.13"),  # XXX: don't specify patch version
```

Comment:
Are the headers needed at runtime?  Can't this be a `BuildDependency` package?

Reply:
They aren't, but I use it to force compatibility: https://github.com/JuliaPackaging/Yggdrasil/blob/5d281759d36224cffd9336be9068ac44649b499b/O/oneAPI_Level_Zero/oneAPI_Level_Zero_Loader/build_tarballs.jl#L39-L42

(so `~1.2` should be fine, but that tried to install `1.2.0` which doesn't exist. is that known?)

Reply:
> (so ~1.2 should be fine, but that tried to install 1.2.0 which doesn't exist. is that known?)

Yes, but you can use
```suggestion
    Dependency("oneAPI_Level_Zero_Headers_jll", v"1.2.13"; compat="~1.2"),  # XXX: don't specify patch version
```
if you want to relax the compat

Reply:
The reason for this is https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/150, which I believe is a reasonable behaviour, but if you want to disentangle compat and build version you can explicitly specify the latter as above